site/content/en/docs/Getting started/helloworld.md

```diff
@@ -15,7 +15,7 @@ with some demo projects including a simple "hello world" workflow.
 3. `cd ~/agc/examples/demo-wdl-project`
 4. `agc context deploy --context myContext`, this step takes approximately 5 minutes to deploy the infrastructure
 5. `agc workflow run hello --context myContext`, take note of the returned workflow instance ID.
-6. Check on the status of the workflow `agc workflow status <workflow-instance-id>`. Initially you will see status like `SUBMITTED` but after the elastic compute resources have been spun up and the workflow runs you should see something like the following: `WORKFLOWINSTANCE    ctx1    9ff7600a-6d6e-4bda-9ab6-c615f5d90734    COMPLETE    2021-09-01T20:17:49Z`
```

Comment:
If this is the first time running any workflow in any context this would be OK but `agc workflow status` will show up to 20 recent runs so including the workflow id helps focus on the correct one.

Reply:
The syntax is still incorrect. We'd want to add the `--run-id` flag here before `<workflow-instance-id>`

Reply:
since this is a hello world example, its better to not make it too complicated and not require user to change command by entering workflow instance id…. Also since its hello world, we can assume it is users first time running this command and hence only one workflow will show up anyway …